charger/wallbe.go

```diff
@@ -55,7 +50,6 @@ func NewWallbeFromConfig(other map[string]interface{}) (api.Charger, error) {
 		Legacy bool
 		Meter  struct {
 			Power, Energy, Currents bool
-			Encoding                string
```

Comment:
Damit startet evcc sofort nicht mehr wenn der Key in der Config ist. Keine gute Idee. Besser interface in "deprecated" Meldung.

Reply:
Verstehe.

Reply:
Schau mal beim loadpoint, da ist das schon so umgesetzt